isis/src/base/apps/framestitch/framestitch.cpp

```diff
@@ -144,7 +139,24 @@ namespace Isis {
 
       frameHeight = evenFrameHeight;
     }
+    int numFrames = evenCube->lineCount() / frameHeight;
+
+    int numLinesOverlap = 0; // The default value
+    if (ui.WasEntered("NUM_LINES_OVERLAP"))
+      numLinesOverlap = ui.GetInteger("NUM_LINES_OVERLAP");
+
+    if (numLinesOverlap % 2 != 0 || numLinesOverlap < 0) {
```

Comment:
Same here, the minimum on the UI already enforces >= 0

Reply:
I think such redundant checks give the reader the impression that a thorough job was undertaken in validating the inputs.